README.md

```diff
@@ -104,6 +127,30 @@ Host your own Chocolatey package repository
 
 #### Parameters
 
+##### `chocolatey_server_app_pool_name`
+Set apppool name used by the chocolatey.server website. Defaults to
+'chocolatey.server'.
+
```

Comment:
I don't think it is, as someone else submitted a merge request previously that used the windows feature module and it didn't look like they needed to supply the default website path.